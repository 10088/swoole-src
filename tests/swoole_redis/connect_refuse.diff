001+ Fatal error: Uncaught Error: Class 'swoole_redis' not found in /src/swoole-src/tests/swoole_redis/connect_refuse.php:4
001- 
002+ Stack trace:
003+ #0 {main}
004+   thrown in /src/swoole-src/tests/swoole_redis/connect_refuse.php on line 4