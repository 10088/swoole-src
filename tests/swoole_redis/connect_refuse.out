Fatal error: Uncaught Error: Class 'swoole_redis' not found in /src/swoole-src/tests/swoole_redis/connect_refuse.php:4
Stack trace:
#0 {main}
  thrown in /src/swoole-src/tests/swoole_redis/connect_refuse.php on line 4